src/components/DuetRoom.tsx

```diff
@@ -31,55 +33,36 @@ const useStyles = makeStyles(theme => ({
   },
   box: {
     flexGrow: 100,
+    position: 'relative',
```

Comment:
Why do we need this?

Reply:
It's for the readybutton to have it's absolute position wrt this element